pkg/cmd/cluster/cluster.go

```diff
@@ -9,9 +9,10 @@ import (
 
 func NewCmdCluster(f *factory.Factory) *cobra.Command {
 	cmd := cobra.Command{
-		Use:  "cluster <command>",
-		Args: cobra.ArbitraryArgs,
-		Long: "View cluster information",
+		Use:   "cluster <command>",
+		Args:  cobra.ArbitraryArgs,
+		Long:  "View cluster information",
+		Short: "View cluster and queue information",
```

Comment:
Perhaps these could be switched? Just by going off of the length of them + their identifier. Non blocker.

Reply:
Agree, changed it to be same as queue might end up as separate command per Jarryd.